x-pack/plugins/security_solution/public/resolver/store/data/visible_entities.test.ts

```diff
@@ -143,7 +144,7 @@ describe('resolver visible entities', () => {
       ];
       const action: ResolverAction = {
         type: 'serverReturnedResolverData',
-        payload: { events, stats: new Map(), lineageLimits: { children: '', ancestors: '' } },
+        payload: { result: mockResolverTree({ events })!, databaseDocumentID: '' },
```

Comment:
@bkimmel @kqualters-elastic @michaelolo24 if you test the data reducer, you might end up needing to use this (until we get around to refactoring)

Reply:
cool, thanks for the fyi